syntax = "proto3";

import "protos/activity_id.proto";
import "protos/squad.proto";

package protos;

message Activity {
  enum State {
    NOT_STARTED = 0;
    FINISHED = 1;
    MILESTONED = 2;
  }

  ActivityID id = 1;
  Squad squad = 2;
  State state = 3;
  string milestone = 4;
}